WebUI/routes/index.js

```diff
@@ -16,32 +16,33 @@ router.post("/generate", function (req, res, next) {
   var docPath = req.body.doc_path;
   var projectName = req.body.project_name;
   var version = req.body.version;
-
+  var uniqueId = Date.now()
```

Comment:
You could use [UUID](https://tools.ietf.org/html/rfc4122) v4 instead of this to get a more reliable unique ID.